.github/workflows/ci.yml

```
@@ -65,4 +67,4 @@ jobs:
 
       # Push Docker image to Docker Hub
       - name: Push Docker image
-        run: docker push kimyujindoji/redunm-app:latest
+        run: docker push kimyujindoji/redunm-app:latest
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)